numpy/polynomial/chebyshev.py

```diff
@@ -1830,6 +1831,77 @@ def chebcompanion(c):
     return mat
 
 
+def chebinterp(func, deg, x=None, args=(), window=None):
```

Comment:
It shouldn't be an alias, and note the Chebyshev is a class. The various functions are left out as they are building blocks, useful if someone else wants to build their own classes. Pulling inheritance based implementations apart to get at the good bits for reuse is a pain. Also, the polynomial classes are not (yet) adapted for more than one dimension.